plugins/tours.py

```diff
@@ -167,3 +170,27 @@ async def waffletour(msg: Message) -> None:
     )
 
     await msg.reply("!viewfaq sibb", False)
+
+
+# --- Tour generation enhancements ---
+
+
+@handler_wrapper(["tournament"])  # hooked on |tournament|create|
+async def tournament_create(conn: Connection, room: Room, *args: str) -> None:
+    if len(args) < 2 or args[0] != "create":
+        return
+
+    tierid = args[1]
+    if tierid.endswith("blitz"):
+        tierid = tierid[:-5]
```

Comment:
In the specific case of blitz formats - only `gen8oublitz` as of now - I think it's more informative to show the rules of the base format.
People mostly care about the banlist and a link to the sample teams thread; they're both absent.
```
>> !tier gen8oublitz
No description found for this Singles Sw/Sh Singles format.
Banlist/Ruleset
Ruleset - [Gen 8] OU, Blitz
```